cmd/rancherd/go.mod

```diff
@@ -8,15 +8,15 @@ replace (
 	github.com/containerd/btrfs => github.com/containerd/btrfs v0.0.0-20181101203652-af5082808c83
 	github.com/containerd/cgroups => github.com/containerd/cgroups v0.0.0-20200531161412-0dbf7f05ba59
 	github.com/containerd/console => github.com/containerd/console v0.0.0-20181022165439-0650fd9eeb50
-	github.com/containerd/containerd => github.com/rancher/containerd v1.4.0-k3s1
+	github.com/containerd/containerd => github.com/k3s-io/containerd v1.4.4-k3s1
```

Comment:
Do you know what this is from ? Is this about the containerD changes that have gone on recently ?

Reply:
The short answer is: we get errors without this.

I don't know what it is related to, but when I saw the error (when running `go mod tidy`), I found a similar replace directive in rancher/rke2. I copied that directive here.

Reply:
this repo got migrated as part of moving k3s to CNCF